src/mare/compiler/serve_definition.cr

```diff
@@ -0,0 +1,133 @@
+require "llvm"
+require "debug"
```

Comment:
Note that this being here breaks the CI process. Here's the error:

```
in src/mare/compiler/serve_definition.cr:2: while requiring "debug": can't find file 'debug'
```

So I think we must remove it.
```suggestion
```